src/app.js

```diff
@@ -123,15 +167,23 @@
 		
 		return this;
 	};
-	
+
 	/**
-	* Change the current page to the requested route
-	* Do nothing if the current page is already the requested route
-	*/
+	 * Change the current page to the requested route
+	 * Do nothing if the current page is already the requested route
+	 * @param {String} obj page requested
+	 * @param {String} previousPoppedUrl
+	 * @this App
+	 */
```

Comment:
We need to find a way to add in the doc the fact that this method will publish App.modules.notify('page.leave', {page: previousPage});

I've never been able to do it, but this is THE MOST important information to have